### PR TITLE
[minor] Add support for Cloud Pak for Data 4.8.x - This will be discarded soon

### PR DESCRIPTION
**Pull request by andrercm**
This PR covers support for Cloud Pak for Data 4.8.x using "MAS managed" approach where we replicate the install/upgrade for CPD 4.8 just like we've been doing for the past CPD releases.

### CODE REVIEW

File: ibm/mas_devops/roles/dro/README.md
Comment (durera) [inline ibm/mas_devops/roles/dro/README.md:1]: Are the updates a bad search/replace?

File: ibm/mas_devops/roles/cp4d/defaults/main.yml [hunk 1]
```diff
@@ -35,4 +35,18 @@ common_services_channel: "{{ lookup('env', 'COMMON_SERVICES_CHANNEL') | default(
 cpd_operators_namespace: "{{ lookup('env', 'CPD_OPERATORS_NAMESPACE') | default('ibm-cpd-operators', true) }}"
 cpd_instance_namespace: "{{ lookup('env', 'CPD_INSTANCE_NAMESPACE') | default('ibm-cpd', true) }}"
 cpd_product_version: "{{ lookup('env', 'CPD_PRODUCT_VERSION') }}"
-cpd_supported_versions: ['4.5.0', '4.5.3', '4.6.0', '4.6.3', '4.6.4', '4.6.6']
+zen_operator_namespace: "{{ (cpd_minor_version is version('4.6','<=')) | ternary('ibm-common-services', cpd_operators_namespace) }}" # depending on the cpd version, zen operator will exist in a different namespace
+
+# ------ CPD 4.8+ specific -----------------------------------------------------------------------------------------------
+cpd_cs_control_namespace: "{{ lookup('env', 'CPD_CS_CONTROL_NAMESPACE') | default('cs-control', true) }}"
+cpd_cpfs_namespace: ibm-common-services
```
Comment (durera) [inline ibm/mas_devops/roles/cp4d/defaults/main.yml:42]: might not be needed ...

File: ibm/mas_devops/roles/cp4d/defaults/main.yml [hunk 2]
```diff
@@ -35,4 +35,18 @@ common_services_channel: "{{ lookup('env', 'COMMON_SERVICES_CHANNEL') | default(
...
+cpd_ibm_licensing_namespace: ibm-licensing
```
Comment (durera) [inline ibm/mas_devops/roles/cp4d/defaults/main.yml:43]: might not be needed

File: ibm/mas_devops/roles/cp4d/templates/catalog_sources/opencloud-operators-4.8.1.yml.j2 [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (durera) [inline ibm/mas_devops/roles/cp4d/templates/catalog_sources/opencloud-operators-4.8.1.yml.j2:1]: Duplicate / split these out

File: ibm/mas_devops/roles/cp4d/templates/catalog_sources/cpd-platform-4.8.1.yml.j2 [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (durera) [inline ibm/mas_devops/roles/cp4d/templates/catalog_sources/cpd-platform-4.8.1.yml.j2:1]: Duplicate / split these out

File: ibm/mas_devops/roles/cp4d/templates/catalog_sources/opencloud-operators-4.8.1.yml.j2 (hunk 3 above)
Comment (andrercm) [reply ibm/mas_devops/roles/cp4d/templates/catalog_sources/opencloud-operators-4.8.1.yml.j2:1]: done

File: ibm/mas_devops/roles/cp4d/templates/catalog_sources/cpd-platform-4.8.1.yml.j2 (hunk 4 above)
Comment (andrercm) [reply ibm/mas_devops/roles/cp4d/templates/catalog_sources/cpd-platform-4.8.1.yml.j2]: done

**Pull request closed without merging**